### PR TITLE
Calculate and display expected charge during 2PT review

### PR DESCRIPTION
**Pull request by Jozzey**
https://eaflood.atlassian.net/browse/WATER-4441

Add functionality to the Review Charge Reference page to allow the user to preview the charge for the Charge Reference.

The preview of the charge will be triggered by a button on the Review Charge Reference details page and will display the result in a banner at the top of the screen.

This is the first of two PRs. In this PR a new Charging Module "request" is being added to enable a request to be sent to the Charging Module to calculate a charge. The next PR will cover adding the functionality to the screens to enable a charge to be calculated for a charge reference.

### CODE REVIEW

File: test/requests/charging-module/calculate-charge.request.test.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+              statusCode: 401,
+              error: 'Unauthorized',
+              message: 'Invalid JWT: Token format not valid',
+              attributes: { error: 'Invalid JWT: Token format not valid' }
+            }
+          }
+        })
+      })
+
+      it('returns a `false` success status', async () => {
```
Comment (Beckyrose200) [inline test/requests/charging-module/calculate-charge.request.test.js:91]: I don't think we use backticks for this. In other unit tests, we do double quotation marks and then singles inside the sentence.

`it("returns a 'false' success status", async () => {`

File: test/requests/charging-module/calculate-charge.request.test.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      it('returns the error in the `response`', async () => {
+        const result = await CalculateChargeRequest.send(transactionData)
+
+        expect(result.response.statusCode).not.to.exist()
+        expect(result.response.body).not.to.exist()
+        expect(result.response.message).to.equal("Timeout awaiting 'request' for 5000ms")
+      })
+    })
+  })
+})
```
Comment (Beckyrose200) [inline test/requests/charging-module/calculate-charge.request.test.js:129]: Need a space here

**Review by Beckyrose200 — CHANGES_REQUESTED** [review]

**Review by Cruikshanks — CHANGES_REQUESTED** [review]

File: test/requests/charging-module/calculate-charge.request.test.js (hunk 1 above)
Comment (Jozzey) [reply test/requests/charging-module/calculate-charge.request.test.js:91]: We have done this in the past, what I have there is a copy and paste from another test. Have a look at this one that @Cruikshanks did `test/requests/charging-module/create-customer-change.request.test.js`. Will check in our dev catch-up this morning to see if we have a standard for doing this kinda thing.

**Review by Cruikshanks — APPROVED** [review]
Could I just ask that you update the PR title before merging? It makes it sound like this is all the work, when in fact we are just adding the supporting charging module API request.

**Review by Beckyrose200 — APPROVED** [review]

**Pull request merged**